test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,33 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        //Empty list
+        assertNoneNull();
+
+        //Only one non-null
+        assertNoneNull(1);
+
+        //Only one null
+        assertSomeNull(null);
```

Comment:
If you hadn't introduced the items == null conditional in Utils.java, the error which would have been raised could be handled by simply casting null as (Object) null.